client/source/js/modules/d3-charts/d3-charts-service.js

```diff
@@ -383,6 +383,8 @@ define(['./module', 'd3', 'underscore', './scale-helpers'], function (module, d3
     }
 
     function drawAxes(scales, options, axesGroup, chartSize) {
+      if (scales === undefined) return;
```

Comment:
disabling was better because empty graph placeholder had a different height due to a missing legend